include/seqan/basic/alphabet_residue.h

```diff
@@ -482,8 +482,8 @@ unknownValueImpl(Rna5 *)
  *
  * @signature typedef SimpleType<unsigned char, Iupac_> Iupac;
  *
- * The ValueSize of <tt>Iupac</tt> is 16.  The nucleotides are enumerated from 0 to 19 in this order: 'U'=0, 'T', 'A',
- * 'W', 'C', 'Y', 'M', 'H', 'G', 'K', 'R', 'D', 'S', 'B', 'V', 'N'=15.
+ * The ValueSize of <tt>Iupac</tt> is 16.  The nucleotides are enumerated from 0 to 15 in this order: 'U'=0, 'A'=1, 'C',
```

Comment:
```suggestion
 * The ValueSize of <tt>Iupac</tt> is 16.  The nucleotides are enumerated from 0 to 15 in this order: '='=0, 'A'=1, 'C',
```
?

Reply:
Oh, I misunderstood. I thought that was a typo. I have no clue what the `=` is supposed to do in that table. AFAIK it's not part of the official IUPAC alphabet. Maybe it's a bug?

Reply:
Tru it is not the official IUPAC but it is the SAM alphabet. (`=` means "equal to the reference base.") So I think the documentation is fine for now as the other IUPAC characters are included.

Reply:
@cpockrandt  Can you update the remaining change :)?